packages/mobile/src/hooks/useOneTimeDrawer.ts

```diff
@@ -0,0 +1,33 @@
+import { useEffect } from 'react'
+
+import AsyncStorage from '@react-native-async-storage/async-storage'
+import { useDispatch } from 'react-redux'
+import { useAsync } from 'react-use'
+
+import type { Drawer as DrawerName } from 'app/store/drawers/slice'
+import { setVisibility } from 'app/store/drawers/slice'
+
+type UseOneTimeDrawerProps = {
+  key: string // AsyncStorage key
+  name: DrawerName
+  disabled?: boolean
+}
+
+export const useOneTimeDrawer = ({
```

Comment:
i wonder if we can even just make a "OneTimeDrawer" even?

Reply:
I bet we can. and we probs eventually should